src/libraries/restore/runtime/runtime.depproj

```diff
@@ -73,6 +79,20 @@
     <Exec Command="chmod +x $(TestHostRootPath)%(DotnetExe.Filename)%(DotnetExe.Extension)" Condition="'$(TargetOS)' != 'Windows_NT' and '$(OS)' != 'Windows_NT'"/>
   </Target>
 
+  <Target Name="SetupRuntimePackNative"
+          Condition="'$(BinPlaceTestRuntimePack)' == 'true' and '$(RuntimeFlavor)' == 'Mono'"
+          AfterTargets="AfterResolveReferences">
+    <Copy SourceFiles="@(RuntimeFiles)"
```

Comment:
Can we instead use `ReferenceCopyLocalPaths` and set `DestinationSubPath`? That way we don't have to add `Copy` tasks and just rely on the SDK to do it for us?

Reply:
`ReferenceCopyLocalPaths` picked up a couple extra files we didn't need (dotnet and some host dylib if I remember correctly).  Wasn't quite sure how to remove it.

Reply:
I believe that was because of the `PackageReferences` in order to run the `System.Data.SqlClient` tests since we don't build that project anymore, we need it to be part of the testhost, so maybe for mono we're still interested on that being part of the `RuntimePack`. 

What you can do, is condition the `PackageReferences` to only be included when necessary, that way your `ReferenceCopyLocalPaths` will be empty. That is where those are coming from. 

Also what you could do if you don't want to mess up with `PackageReferences` you can add the files you don't care to to `FilesToExclude` or you could also add the packages to `PackageToExclude` item for packages you don't care.

Then you can use `ReferenceCopyLocalPaths`

Reply:
Removed Copy usage by instead using `NativeBinPlaceItem`